docs/Proxmox.md

```diff
@@ -0,0 +1,158 @@
+# Using Proxmox with TopoMojo
+
+[Proxmox Virtual Environment (PVE)](https://pve.proxmox.com/wiki/Main_Page) is an open source server virtualization management solution based on QEMU/KVM and LXC. TopoMojo can be configured to use a PVE cluster to deploy QEMU virtual machines rather than it's traditional VMware based virtual machines.
+
+## Proxmox Setup
+
+There are a few things you wil need to do within Proxmox to prepare it for use with TopoMojo.
+
+### Installation
+
+- Install Proxmox on one or more nodes.
+    - Add all of the nodes that you want to be used by TopoMojo to a single Proxmox cluster.
+
+### Generate an Access Token
+
+TopoMojo requires a Proxmox Access Token in order to authenticate with the Proxmox API.
+
+- From the Proxmox Web UI, generate an API Token by clicking on Datacenter and navigating to Permissions -> API Tokens.
+    - Ensure Privilege Separation is unchecked if you want to use the privileges of the token user. Otherwise, you will need to select individual permissions to give to the token.
+    - Copy the Secret and the Token ID. This will need to be added to appsettings later.
+
+### Create an SDN Zone
+
+TopoMojo uses Proxmox's Software Defined Networking (SDN) feature to manage the networks of the virtual labs. You will need to create an SDN Zone in Proxmox and configure TopoMojo to use it.
+
+- In the Proxmox Web UI navigate to Datacenter -> SDN -> Zones.
+- Add a new VXLAN Zone.
+    - VXLAN is the only type currently supported by TopoMojo.
+    - The ID is the name you want to use for this Zone. You will need it when configuring TopoMojo's appsettings.
+    - Under Peer Address List, enter a comma separated list of the IP Addresses of all of the nodes in your cluster.
+        - If you add a new node to your cluster, you will need to add it to the SDN Zone as well.
+
+### Configure NGINX
+
+You will need to configure a reverse proxy on the node that TopoMojo will communicate with in order to access the API and allow viewing of consoles. Instructions for doing this with NGINX are provided below.
```

Comment:
I know we do have to do this, but I forget exactly why. It's CORS-related, right? If you remember why off the top of your head, might be worth including here so people know why they need it. 🤷🏻

Reply:
You're thinking of NGINX settings for development, which is separate from the Proxmox reasons. Will add a blurb about the Proxmox-specific reasoning.